recipe/style.css

```diff
@@ -0,0 +1,82 @@
+body{
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  min-height: 100vh;
+  background-image: url("img/dish.jpg");
+  background-size: cover;
+  margin: 0;
+  padding: 25px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #333;
+}
+
+.section-blocks {
+  background-color: rgba(255,255,255,0.8);
+  padding: 15px 30px;
+  border-radius: 10px;
+  box-shadow: 2px 2px 5px rgba(0,0,0,0.2);
+  width: 30%;
+  max-width: 400px;
+}
+
+@media (max-width: 996px) {
+  .section-blocks {
+    margin: 0 auto 15px;
+    width: 100%;
+  }
+}
+
+.top-section {
+  display: flex;
+  justify-content: space-between;
+}
+
+@media (max-width: 996px) {
```

Comment:
Usually, we should stick to the standard [breakpoints](https://getbootstrap.com/docs/5.0/layout/breakpoints/) unless there is a good reason not to use it